### PR TITLE
Refactored : Added All components in App file & include Providers in …

### PR DESCRIPTION
**Pull request by Rishi05051997**
Refactored : Added All components in App file & include Providers in index file

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,82 @@
...
+import { Signup, ChangePassword, Login } from "./components/Auth/index";
+import Cart from "./components/Cart/Cart";
+import Wishlist from "./components/Wishlist/Wishlist";
+import ProductDetails from "./components/Products/ProductDetails/ProductDetails";
 import { Routes, Route } from "react-router-dom";
-import { fetchCategories } from "./services/index"
 import Home from "./components/Home/Home";
+import Profile from "./components/Profile/Profile";
+import ProductsListing from "./components/Products/ProductsListing";
+import { fetchAllProducts, initUserCart, fetchCategories, initUserAddress, initializeUserWishlist, updateCart } from "./services/index"
```
Comment (rahulyadav139) [inline src/App.js:13]: I think you don't need to specify the index here. "./services" path will work.

**Review by rahulyadav139 — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,42 +1,82 @@
...
 import { useData } from "./context/data-context";
-import Nav from "./components/Layout/Nav";
-import "./styles/main.css";
+import { useAuthContext } from "./context/auth-context";
+import axios from "axios"
+import { Toast } from "./components/Toast/Toast";
+import { Order } from "./components/Private/Order/Order";
```
Comment (TejasShekar) [inline src/App.js:18]: You can create an index.js in components folder, do a named export and here you can import them using `./components` . No need of such long path names.

**Review by TejasShekar — COMMENTED** [review]
Just one recommendation. Rest looks good to merge.

**Review by TejasShekar — APPROVED** [review]

File: src/App.js (hunk 2 above)
Comment (Rishi05051997) [reply src/App.js:18]: As a refactoring I noted this down

File: src/App.js (hunk 1 above)
Comment (Rishi05051997) [reply src/App.js:13]: Yes done

**Pull request merged**